apps/judicial-system/web/src/shared-components/Header/Header.spec.tsx

```diff
@@ -26,4 +37,31 @@ describe('Header', () => {
     // Assert
     expect(history.location.pathname).toEqual(Constants.REQUEST_LIST_ROUTE)
   })
+
+  test('should direct admin users to user overview route when user click the logo', async () => {
+    // Arrange
+    const history = createMemoryHistory()
+    history.push(Constants.USER_NEW_ROUTE)
+    const { getByTestId } = render(
+      <MockedProvider mocks={mockAdminQuery} addTypename={false}>
+        <Router history={history}>
+          <UserProvider>
+            <Header pathname={Constants.USER_LIST_ROUTE} />
+          </UserProvider>
+        </Router>
+      </MockedProvider>,
+    )
+
+    // wait for login to complete
+    await screen.findByTestId('logout-button')
```

Comment:
A better way of doing this is `screen.findByRole('button', { name: user.name })`. According to https://testing-library.com/docs/queries/about/, you should always try to use *ByRole whenever you can.

Reply:
ok